config/chime_science_run_gpu.yaml

```diff
@@ -1548,12 +1549,12 @@ eigcalc:
 
   # Masking config
   bands_filled: [ [0, 30], [251, 262] ]
-  exclude_inputs: [
-    46,   84,  107,  136,  142,  256,  257,  319,  348,  370,  550,
-    551,  579,  638,  688,  742,  768,  784,  807,  855,  944,  960,
-    971, 1005, 1010, 1020, 1023, 1058, 1166, 1225, 1280, 1281, 1285,
-    1314, 1380, 1521, 1523, 1543, 1642, 1687, 1738, 1792, 1794, 1910,
-    1912, 1943, 1945, 1982, 1984, 1987, 2032, 2034 ]
+  exclude_inputs: [  32,   46,   84,  107,  136,  142,  256,  257,  319,  348,  357,
```

Comment:
where do you get 357 and 369 from and why weren't they included before? I don't just know.

Reply:
They were in the default bad inputs list, but not in the exclude_inputs list for the eigenvalue code, so I added them at the same time.